src/app/tab-events/tab-events.page.html

```diff
@@ -0,0 +1,92 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-title>Eventos</ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content>
+  <ng-container *ngIf="majorEvents; else loading">
+    <h1 *ngIf="majorEvents.length === 0" class="ion-text-center">Nenhum evento encontrado</h1>
+
+    <div *ngIf="majorEvents.length > 0" class="event-container">
+      <ng-container *ngFor="let event of majorEvents">
+        <ion-card>
+          <ion-card-header>
+            <ion-card-title>{{event.name}}</ion-card-title>
+          </ion-card-header>
+
+          <ion-card-content>
+            <p>{{event.description}}</p>
+          </ion-card-content>
+
+          <div class="card-section">
+            <div>
+              <ion-icon name="calendar-outline"></ion-icon>
+              Datas
+            </div>
+
+            <ion-item lines="none">
+              Evento
+              <span slot="end">
+                {{ getDateFromTimestamp(event.dateStart) | date: 'dd/MM' }}
+                {{ event.dateEnd && '- ' + (getDateFromTimestamp(event.dateEnd) | date: 'dd/MM') }}
+              </span>
+            </ion-item>
+
+            <ion-item lines="none">
+              Inscrição
+              <span slot="end">
+                {{ getDateFromTimestamp(event.subscriptionDateStart) | date: 'dd/MM' }}
+                {{ event.subscriptionDateEnd && '- ' + (getDateFromTimestamp(event.subscriptionDateEnd) | date: 'dd/MM') }}
+              </span>
+            </ion-item>
+          </div>
+
+          <div class="card-section">
+            <div>
+              <ion-icon name="wallet-outline"></ion-icon>
+              Valores
+            </div>
+
+            <ng-container *ngIf="isObject(event.price); then multiPrice else uniquePrice"></ng-container>
+
+            <ng-template #uniquePrice>
+              <ion-item lines="none">
+                Preço único
+                <span slot="end">{{ (event.price / 100) | currency:'BRL' }}</span>
+              </ion-item>
+            </ng-template>
+
+            <ng-template #multiPrice>
+              <ion-item lines="none">
+                Alunos da UNESP
+                <span slot="end">{{ (event.price.priceStudents / 100) | currency:'BRL' }}</span>
+              </ion-item>
+
+              <ion-item lines="none">
+                Alunos
+                <span slot="end">{{ (event.price.priceOtherStudents / 100) | currency:'BRL' }}</span>
+              </ion-item>
+
+              <ion-item lines="none">
+                Professores
+                <span slot="end">{{ (event.price.priceProfessors / 100) | currency:'BRL' }}</span>
+              </ion-item>
+            </ng-template>
+          </div>
+
+          <div class="card-button-section">
+            <ion-button>
+              INSCREVA-SE
+              <ion-icon name="arrow-forward-outline" slot="end" style="font-size: 16px"></ion-icon>
+            </ion-button>
+          </div>
+        </ion-card>
```

Comment:
Está faltando o botão "saiba mais"
```typescript
interface majorEvent {
  button?: {
    text?: string,
    url: string
  };
}
```
	
`uid.button.text` - Se o texto for indefinido, o padrão é "saiba mais"
`uid.button.url`

Pode deixar o botão de "inscreva-se" no estilo que está, mas coloque o "saiba mais" no estilo abaixo, para vermos como fica a harmonia entre os dois.

<img width="521" alt="image" src="https://user-images.githubusercontent.com/12040397/188527601-c3a9465c-b0bc-43ad-87b7-6a5b7753c400.png">